vm/devices/net/net_consomme/consomme/src/udp.rs

```diff
@@ -169,6 +187,10 @@ impl<T: Client> Access<'_, T> {
             &checksum.caps(),
         )?;
 
+        if udp.src_port == 50015 {
```

Comment:
what's special about this port?

Reply:
yikes, that shouldn't have been left in...

Reply:
haha oops.